docs/examples/lambda-aws/serverless.yml

```diff
@@ -0,0 +1,28 @@
+service: app
+
+provider:
+    name: aws
+    region: us-east-1
+    environment:
+        MONGODB_URI: ${env:MONGODB_URI}
+
+plugins:
+    - ./vendor/bref/bref
+    - ./vendor/bref/extra-php-extensions
+
+functions:
+    api:
+        handler: index.php
+        description: ''
+        runtime: php-83-fpm
+        timeout: 28 # in seconds (API Gateway has a timeout of 29 seconds)
+        events:
+            -   httpApi: '*'
+        layers:
+            - ${bref-extra:mongodb-php-83}
+
+# Exclude files from deployment
+package:
+    patterns:
+        - '!node_modules/**'
+        - '!tests/**'
```

Comment:
Not necessary, this are part of the default setup. Should I remove them or keep as an example?

Reply:
Why would `node_modules` be here at all? Also, I assume the `tests/` path here has nothing to do with the top-level tests directory, and this is all relative to the `examples/lambda-aws` path where this YAML and the `index.php` file exist.

If these truly server no purpose, I'd propose removing them. Optionally leave a YAML comment pointing to docs for `package.patterns` in case users want to add their own stuff.

Reply:
Since we're just creating a small example, I'm in favour of removing all unnecessary configuration. Adding it just leads to people keeping it around without knowing why they do so.

Reply:
```suggestion
```